### PR TITLE
CRM457-2203: XML import

### PR DESCRIPTION
**Pull request by patrick-laa**
## Description of change
Add a new form to upload an XML file and that, if the file contains disbursements and work items in the expected format, will create a new claim and import details from the file.

[Link to relevant ticket](https://dsdmoj.atlassian.net/browse/CRM457-2203)

### CODE REVIEW

File: app/services/nsm/import_service.rb
Comment (elken) [inline app/services/nsm/import_service.rb:17]: Will this validate it against the included schema?

**Review by elken — COMMENTED** [review]
Once the q is answered, this is fine to approve

File: app/services/nsm/import_service.rb
Comment (patrick-laa) [reply app/services/nsm/import_service.rb:17]: No, because while the sample XML file we got given contains a schema, the real-life XML files we've been given don't, and I wouldn't trust them to adhere to it. So instead we're trying to be as permissible as possible - if it contains some information that we can read, we read that information (and then prompt the user to check it)

Comment (elken) [reply app/services/nsm/import_service.rb:17]: Always hesitant to handle XML we can't fully validate, but if there's no alternative I guess we have little choice.

**Review by elken — APPROVED** [review]

**Comment by ivanELEC** [comment]
Pending a review of the visual side of things, LGTM!

**Pull request merged**
